About digipres (#24)
artefactual/artefactual.github.io: docs/digipres.rst

About digital preservation
==========================

At Artefactual, we work in the context of archival standards.
This context informs the decisions we make for the software we
develop and maintain.

The resources listed here can be specific to our own context, since
there are a diverse, broad range of archival traditions in the
world. We believe that the list presented here is a good first step
to get started in the digital preservation world. This list has been
curated for beginners.

Digital preservation is for everyone
------------------------------------

This document is on resources about digital preservation and archives
in general. The goal is to help someone new to this world get
familiar with the concepts and ideas, not to replace a formal
education on the topic.
This is not meant to be exhaustive.

What is digital preservation?
-----------------------------

Digital preservation is the set of actions, standards and processes
involved in making digital or digitalized content available in the

> future.

Reviewer: suggestions: "digitized" instead of "digitalized" and add "and authentic" to available.

"Digital preservation is the set of actions, standards and processes involved in making digital or digitized content available and authentic in the future."

> Digital preservation might involve standard IT storage solutios,

Reviewer: solutions

> Digital preservation might involve standard IT storage solutios,

Author: ```suggestion
Digital preservation might involve standard IT storage solutions,
```

Standards
---------

Our work is informed both by archival content standards and by
metadata standards.

Metadata standards
``````````````````

Software developed at Artefactual is informed by the digital
preservation standards, such as `PREMIS`_ and
`METS`_. Both these websites contain learning materials,
such as this `PREMIS list of tutorials`_
as well as `suggested readings for METS`_.

Archivematica documentation has pages on
`how PREMIS is supported in the software`_,
and similarly for `METS in Archivematica`_.

Archival content standards
``````````````````````````

Some of the archival contents standards we are interested in
includes the Rules for Archival Description, or `RAD`_, the
`DACS`_ and `ISAD-G`_. Other standards might be
of interest which are not listed here.

Technical challenges
--------------------

If you are curious about the technical challenges of preserving data
and its metadata for the long term, consider reading about the
`challenges faced by the CERN Digital Memory Platform`_,
such as scaling, hardware failures, losing data because of software
errors, and other ways that digital preservation can fail to happen.
This thesis make use of Archivematica.

> Learning to code

Author: ```suggestion
Software for digital preservation
```

----------------

> 

Reviewer: Would the audience for this not already know how to code?

Author: I guess it could be rephrased... let me see.

Author: I added suggestions for lines 83 and 86 - let me know!

> You would like to learn to program, and you are wondering which

Author: ```suggestion
You are wondering which software are used in digital preservation? Which
```
